vtex-apps/auth-challenge
# Add Trade Policy Condition Check

#### What does this PR do? \*

Instead of just checking weather the user logged in or not, now this challenge looks for the trade policy attached to the user, and renders the child blocks based on it.

#### How to test it? \*

An example has been provided in the README using `product-description` block. Make sure that the B2B setup is complete before testing the feature.
